### ISSUE TITLE
netcdf: build failing with no rule to make target libcmdr

### ISSUE THREAD

**Issue by brucellino** [SouthAfricaDigitalScience/netcdf-deploy#1]
From build 10, we are seeing the build failing with :

```
make[2]: *** No rule to make target `../libcdmr/libnccr.la', needed by `libnetcdf.la'.  Stop.
```

**Comment by brucellino** [comment]
This wasn't closed by removing the netcdf4 stuff :unamused: We were using `--enable-cmdremote` which I removed in 9845f9f which may address the issue.

**Comment by brucellino** [comment]
netcdf4 stuff was not to  blame - rather the option to build remote commands was removed. This may be necessary in the future, but for now to pass builds we'll remove it.

**Comment by brucellino** [comment]
Fixed in 5ec9d169 and build 14 passed.

**Comment by brucellino** [comment]
included in fastrepo 47
